### PR TITLE
T115 include bvf

### PR DESCRIPTION
**Pull request by jnoelke**
Code and tests

- bvf is calcutated within the simulation and part of the hdf5 output
- bvf calculation is tested for various scenarios
-  for all tissue types that include bvf and oxy we tested concistency between intially assigned values and calculations (We put it as part of TestCoreAssumptions but might also go somewhere else.)
- For the tissue composition tests we added dummy values that might need to be adapted (another new issue?)

### CODE REVIEW

File: simpa/utils/calculate.py [hunk 1]
```diff
@@ -34,6 +34,30 @@ def calculate_oxygenation(molecule_list):
...
+    """
+    :return: the blood volume fraction value between 0 and 1, or 0, if oxy and deoxy not present.
+    """
+    hb = None
+    hbO2 = None
+
+    for molecule in molecule_list:
+        if molecule.spectrum.spectrum_name == "Deoxyhemoglobin":
+            hb = molecule.volume_fraction
+        if molecule.spectrum.spectrum_name == "Oxyhemoglobin":
```
Comment (frisograce) [inline simpa/utils/calculate.py:47]: elif

File: simpa_tests/automatic_tests/tissue_library/test_core_assumptions.py [hunk 2]
```diff
@@ -19,10 +21,39 @@ def test_volume_fractions_sum_to_less_or_equal_one(self):
             self.assertAlmostEqual(total_volume_fraction, 1.0, 3,
                                    f"Volume fraction not 1.0 +/- 0.001 for {method_name}")
 
+    def test_bvf_and_oxygenation_consistency(self):
+        # blood_volume_fraction (bvf) and oxygenation of tissue classes defined
+        #         # as input have to be the same as the calculated ones
```
Comment (frisograce) [inline simpa_tests/automatic_tests/tissue_library/test_core_assumptions.py:26]: 2 '#'

File: simpa_tests/automatic_tests/tissue_library/test_core_assumptions.py [hunk 3]
```diff
@@ -6,7 +6,9 @@
 from simpa.utils import TISSUE_LIBRARY
 from simpa.utils.libraries.tissue_library import TissueLibrary
```
Comment (frisograce) [inline simpa_tests/automatic_tests/tissue_library/test_core_assumptions.py:7]: remove from simpa.utils.libraries.tissue_library import TissueLibrary

**Review by frisograce — COMMENTED** [review]
Please check how to save bvf in hdf5. It seems not to appear in the output.

File: simpa_tests/automatic_tests/tissue_library/test_core_assumptions.py (hunk 3 above)
Comment (jnoelke) [reply simpa_tests/automatic_tests/tissue_library/test_core_assumptions.py]: I think without further changes in the code we cannot just remove the import

**Pull request merged**